Services/Devices.cs

```diff
@@ -49,6 +68,12 @@ public class Devices : IDevices
         // the simulation. When the simulation is running the registry is not used anymore.
         private const uint REGISTRY_LIMIT_REQUESTS = 1000;
 
+        // When working with batches, this is the max size
```

Comment:
super nit: "..., this is the max size that the batch insert and delete APIs allow"

Reply:
👍